### DISCUSSION TITLE
What is the Zed lingo/terminology for Sublime-like "projects"/workspaces?

### DISCUSSION THREAD

**Discussion by TSSlade** [zed-industries/zed#17278] [Feature Requests]
In Sublime, I can configure (and then persist) collections of folders that are associated with a given project. I can then easily switch from one project (say, my grad school essays) to another one (say, my quantified self dashboard) from the command palette.

I can't figure out how to do something like this in Zed. Is it just not supported? Or am I using the wrong terms to search for it? 

- "Project" in Zed land seems to just mean "whatever folder I am currently in", and I seem only to be able to navigate (via alt+command+o) to the most recently-opened folder, rather than any folder from an arbitrarily distant past coding session.
- "Workspace" seems to refer to...the layout of the editor interface? I think?

**Comment by thataboy** [comment]
It's also called a project. But zed doesn't let you explicitly save a project and reload it later like sublime. It creates one when you open a folder. In the project pane, you can right click and add more folders to it. When you close the window, it saves the folders and opened files. You can reopen later with File > Open recent (if you have more than one folder in the project, they'll appear as folder1, folder2...) I don't know how many recent projects it will retain.

One thing I don't like about zed is it will not reopen files that don't belong to any folders in the project. So if you open some random files in the ~/Downloads folder, zed won't reopen it next time, unless you add ~/Downloads to your project. This makes it less useful as an all purpose text editor, IMHO.

**Reply by TSSlade** [reply]
Gotcha. I guess it just can't do what I want. Bummer. Thanks for clarifying!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
